src/Plugin.php

```diff
@@ -70,6 +70,9 @@ public static function init() {
     // Run after daggerhart-openid-connect-generic (99).
     add_filter('logout_redirect', __CLASS__ . '::logout_redirect', 100);
```

Comment:
Since the time we deployed this PR, the previously working logout redirect to the SSO does not work anymore. Instead, the user ends up on the page `/wp-login.php` of WordPress, which should not ever appear for front-end users.  This causes the user to not get logged out of the other applications anymore.

I suspect the newly implemented hook `check_admin_referer` is invoked before the `logout_redirect` hook, or not invoked at all anymore with this PR.

Note that the daggerhart-openid-connect-generic plugin also implements the `logout_redirect` hook.